.gitlab-ci.yml

```diff
@@ -358,6 +322,16 @@ pkg:opam-split:
     OPAM_VARIANT: "+flambda"
   only: *full-ci
 
+pkg:opam:
+  extends: .pkg:opam-template
+
+pkg:opam:native:
+  extends: .pkg:opam-template
+  before_script:
+    - opam install -y coq-native
+  after_script:
+    - echo ""
```

Comment:
??

Reply:
Seems related to my comment above: https://github.com/coq/coq/pull/15560#discussion_r884241937

Reply:
Yes, I think I left this as a placeholder for adding that test.